pages/linux/fdisk.md

```diff
@@ -1,6 +1,8 @@
 # fdisk
 
 > A program for managing partition tables and partitions on a hard disk.
+> See also: `partprobe`.
+> More information: <https://manned.org/fdisk>.
```

Comment:
Hey @aherst, the link for `fdisk` is already added in #6076
```suggestion
```

Reply:
Yes, this will stop a conflict between nothing. just makes future easier for others.